src/ScrollSync.jsx

```diff
@@ -142,6 +138,8 @@ export default class ScrollSync extends Component {
   }
 
   render() {
-    return React.Children.only(this.props.children)
+    return (<ScrollSyncContext.Provider value={this.getContextValue()}>
+      {React.Children.only(this.props.children)}
+    </ScrollSyncContext.Provider>)
```

Comment:
Why is parentheses needed? I think it can be:

```suggestion
    return <ScrollSyncContext.Provider value={this.getContextValue()}>
      {React.Children.only(this.props.children)}
    </ScrollSyncContext.Provider>
```

Reply:
Eslint exends from `okonet` that extend from `airbnb` rules that has [react/jsx-wrap-multilines](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md).

But without parentheses is cleaner so i disabled this rule.

Changes: https://github.com/okonet/react-scroll-sync/pull/74/commits/51cabd7d3048a039ae1d4e9c05392ce6df66600b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R39